docs/en/observability/cloud-monitoring/aws/monitor-aws-esf.asciidoc

```diff
@@ -0,0 +1,159 @@
+    }
+  ]
+}
+----
+
+. Go to the properties of the S3 bucket containing the VPC flow logs and enable event notification.
+
+For more details, refer to the AWS documentation on how to https://docs.aws.amazon.com/AmazonS3/latest/userguide/ways-to-add-notification-config-to-bucket.html[Configure a bucket for notifications].
+
+[discrete]
+[[esf-step-four]]
+=== Step 4: Install the Elastic AWS integration 
+
+{kib} offers prebuilt dashboards, ingest node configurations, and other assets that help you get the most value out of the logs you ingest. 
+
+. Go to *Integrations* in {kib} and search for AWS. 
+. Click the AWS integration, select *Settings* and click *Install AWS assets* to install all the AWS integration assets.
+
+[discrete]
+[[esf-step-five]]
+=== Step 5: Create a new S3 bucket to store the configuration file
+
+Note that you can store more than one configuration file.
+
+. In the https://s3.console.aws.amazon.com/s3[AWS S3 console], click *Create* bucket. Give the bucket a name and specify the region where you want it deployed.
++
+IMPORTANT: Make sure you create the S3 bucket in the same region as the bucket containing VPC flow logs.
+
+[discrete]
+[[esf-step-six]]
+=== Step 6: Create a configuration file to specify the source and destination  
+
+Elastic Serverless Forwarder uses the configuration file to know the input source and the Elastic connection for the destination information.
+
+. In Elastic Cloud, from the AWS Integrations page click *Connection details* on the upper right corner and copy your Cloud ID. 
+. Create an encoded API key for authentication. 
+
+You are going to reference both the Cloud ID and the newly created API key from the configuration file. Here is an example:
+
+[source,yml]
+----
+inputs:
+  - type: "s3-sqs"
+    id: "<your-sqs-queue-arn>"
+    outputs:
+      - type: "elasticsearch"
+        args:
+          cloud_id: "<your-cloud-id>"
+          api_key: “<your-api-key>>"
+----
+
+. Upload the configuration file you have just created to the S3 bucket you created at step 5.
+
+[discrete]
+[[esf-step-seven]]
+=== Step 7: Ingest VPC flow logs into Elastic
+
+Deploy the Elastic Serverless Forwarder from AWS SAR and provide appropriate configurations for the Lambda function to start ingesting VPC flow logs into Elastic.
+
+. From the Lambda console select *Applications* and click *Create Application*.
+. From the *Serverless application*, sleect *elastic-serverless-forwarder*.
+. On the *Review, configure and deploy* page, fill in the following fields:
++
+- *ElasticServerlessForwarderS3Buckets*: Specify the ARN of the S3 Bucket you created at step 1 where the VPC Flow Logs are sent.
++
+- *ElasticServerlessForwarderS3ConfigFile*: Specify the URL of configuration file in the format "s3://bucket-name/config-file-name".
++
+- *ElasticServerlessForwarderS3SQSEvents*: Specify the S3 SQS Notifications queue used as the trigger of the Lambda function. The value is the ARN of the SQS Queue you created on step 3.
```

Comment:
Inconsistency nit: On this line, you say "on step 3". On line 146, you say "at step 1".